modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,94 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling errors.
+:keywords: autocorrect, capitalization, misprint, spell, spelling, spellcheck, typo, typographical error
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tierthree
+
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
```

Comment:
need to add modules/ROOT/partials/misc/admon-requires-6.2v.adoc

Reply:
The current path is correct.

Reply:
@vpyshnenko, can you also please mark this request as addressed?